test/integration/conftest.py

```diff
@@ -123,3 +123,10 @@ def skip_py2_containers(request, tag):
 def ecr_image(account_id, docker_base_name, tag, region):
     return '{}.dkr.ecr.{}.amazonaws.com/{}:{}'.format(
         account_id, region, docker_base_name, tag)
+
+
+@pytest.fixture(autouse=True)
```

Comment:
This was added in a subsequent commit to aid other open PRs, will you be able to remove this fixture?